tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -10,6 +10,10 @@
           {
             "type": "object",
             "properties": {
+              "recoverybucketoverride": {
+                "description": "The user specified recovery S3 bucket to store recovered files",
+                "type": ["string", "null"]
+              },
```

Comment:
Might be a separate card, but we should probably document this workflow better in recovery-workflow.mdx.
From the point-of-view of a non-Cumulus integrator, it's not clear what properties should be set and how to set them.